web/src/features/filtering/components/ActiveFilters.tsx

```diff
@@ -0,0 +1,68 @@
+import { FilterBadge } from '@/components/ui/badge';
+import { useNavigate } from '@tanstack/react-router';
+import { FC, useCallback } from 'react';
+
+interface ActiveFilterProps {
+  filterId: string;
+  value: string;
+  isArray?: boolean;
+}
+
+type SearchParams = {
+  [key: string]: any;
+};
+
+const HIDDEN_FILTERS = ['pageSize', 'pageNumber'];
+const FILTER_LABELS = new Map<string, string>([
+  ['status', 'Observer status'],
```

Comment:
- we should extract this keys in a enum/class so we can not rely on magic strings
- use the key from enum/class in `~/features/filtering/components/ObserverStatusSelect.tsx` line 26
- key name should be `monitoringObserverStatus` since we have status on many entities